### ISSUE TITLE
Transparent background-color styling throwing "Insufficient Text Color Contrast" error

### ISSUE THREAD

**Issue by kywoodland** [ucfopen/UDOIT#908]
The following code produces an "Insufficient Text Color Contrast With the Background" error even if there is no contrast issue:
`<p style="background-color: transparent;">`
Removing the `background-color` styling gets rid of the error but it's impossible to see it in the Rich Content Editor. UDOIT cannot change the color using the arrows to adjust value up and down in the UFIXIT editor (nothing happens if you use it), but it can successfully change the background color using the color picker drop-down.

**Comment by dmols** [comment]
Updating here. The PR above was closed because it involves a solution that skips instructor approval, and that's an essential part of the solution we want to implement. Will be revisiting this issue in the future.
